test/unit/specs/components/Lend/LoanSearch/LoanSearchSectorFilter.spec.js

```diff
@@ -0,0 +1,40 @@
+import { render } from '@testing-library/vue';
+// eslint-disable-next-line import/no-unresolved
```

Comment:
I think you can remove this `eslint`?

Reply:
I'll resolve this in my branch, merging this PR so I can complete my current changeset.